app/server/api/synchronization_filter_api.py

```diff
@@ -24,8 +24,77 @@ def post(self):
 
         return make_response(jsonify(synchronization_filter_schema.dump(transaction_filter).data)), 201
 
+class SynchronizationFilterMetricsAPI(MethodView):
+    # Gets metrics regarding the status of third party sync
+    @requires_auth(allowed_roles={'ADMIN': 'admin'})
+    def get(self):
+        """
+        Gets status metrics for third party transaction sync
+        Params:
+        get_failed_block_fetches - 'true' or 'false', will return a list of blocks that failed being fetched (default false)
+        get_failed_callbacks - 'true' or 'false', will return a list of callbacks which have failed (default false)
+        """
+        get_failed_block_fetches = request.args.get('get_failed_block_fetches', 'false').lower() in ['true', '1']
+        get_failed_callbacks = request.args.get('get_failed_callbacks', 'false').lower() in ['true', '1']
+
+        resp = {}
+        if get_failed_block_fetches:
+            resp['failed_block_fetches_by_contract'] = bt.get_failed_block_fetches()
+
+        if get_failed_callbacks:
+            resp['failed_callbacks_by_contract'] = bt.get_failed_callbacks()
+
+        resp['metrics'] = bt.get_third_party_sync_metrics()
+        return make_response(jsonify(resp)), 201
+
+class SynchronizationFilterRefetchAPI(MethodView):
+    @requires_auth(allowed_roles={'ADMIN': 'admin'})
+    def post(self):
+        """
+        Refetches a given range of blocks for third party transaction sync
+        Params:
+        filter_address - Address associated with a token.
+        floor - Where to start fetching blocks
+        ceiling - Where to finish fetching blocks
+        """
+        filter_address = request.args.get('filter_address')
+        floor = int(request.args.get('floor'))
+        ceiling = int(request.args.get('ceiling'))
+        result = bt.force_fetch_block_range(filter_address, floor, ceiling)
+        return make_response(jsonify({ 'status': result })), 201
+
+class SynchronizationFilterRecallWebhookAPI(MethodView):
+    @requires_auth(allowed_roles={'ADMIN': 'admin'})
+    def post(self):
+        """
+        Forces a recall of the webhook for a given transaction
+        Params:
+        transaction_hash - Hash for the transaction you want to resync!
+        """
+        transaction_hash = request.args.get('transaction_hash')
+        bt.force_recall_webook(transaction_hash)
+        
+
 synchronization_filter_blueprint.add_url_rule(
     '/synchronization_filter/',
     view_func=SynchronizationFilterAPI.as_view('synchronization_filter_view'),
     methods=['POST']
 )
+
+synchronization_filter_blueprint.add_url_rule(
+    '/synchronization_filter/status/',
+    view_func=SynchronizationFilterMetricsAPI.as_view('synchronization_filter_metrics_view'),
+    methods=['GET']
+)
+
+synchronization_filter_blueprint.add_url_rule(
+    '/synchronization_filter/refetch_block_range/',
```

Comment:
Should these follow the same pattern as the Blockchain Transaction RPC for consistency?
https://github.com/teamsempo/SempoBlockchain/blob/master/app/server/api/blockchain_transaction_api.py#L10

Reply:
My thinking here is that these calls don't really have to do with a `blockchain_transaction` per-se. What if we rename such that `/synchronization_filter/recall_webhook` becomes `/synchronization_filter_rpc/recall_webhook`?

Reply:
oh what i meant was the pattern of 
```
 post_data = request.get_json()

        call = post_data.get('call')

        if call == 'RETRY_TASK':
```